### PR TITLE
Developer doc

### PR DESCRIPTION
**Pull request by Aswin40**
Added developer doc folder,Added steps to set up the front-end code development environment.

### CODE REVIEW

**Comment by JithinKS97** [comment]
Can you update this pull request towards dev branch instead of master?

**Comment by JithinKS97** [comment]
Can you update this pull request towards the dev branch instead of master?

File: developer_docs/Intallation.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Setting up the development environment
+
+Follow the below steps to setup the development environment which you need before  contributing to the project
+
+## Prerequesites
+
+The main libraries used in the project are ReactJS,NextJS and NodeJS,NPM as the dependency management tool.
```
Comment (Anu-Asok) [inline developer_docs/Intallation.md:7]: @JithinKS97 We are using yarn for package management, right?

Comment (JithinKS97) [reply developer_docs/Intallation.md:7]: I faced some issues with yarn when trying to install the fabric canvas library. So switched to npm.

Comment (Anu-Asok) [reply developer_docs/Intallation.md:7]: Okay

Comment (JithinKS97) [reply developer_docs/Intallation.md:7]: @Anu-Asok Should we also consider adding instructions for setting up the back end dev environment as well in the doc?

Comment (Anu-Asok) [reply developer_docs/Intallation.md:7]: @JithinKS97 I’ll add the instructions for backend dev env setup in a separate doc once I’m done creating basic GraphQL API’s. Let’s not mix up things.

**Comment by Aswin40** [comment]
> Can you update this pull request towards dev branch instead of master?

okay

**Pull request merged**